jekyll/_cci2/browser-testing.md

```diff
@@ -158,7 +162,39 @@ workflows:
 ```
 {% endraw %}
 
-For more detailed information about the Sauce Labs orb and how you can use the orb in your workflows, refer to the [Sauce Labs Orb](https://circleci.com/developer/orbs/orb/saucelabs/sauce-connect) page in the [CircleCI Orbs Registry](https://circleci.com/developer/orbs).
+For more detailed information about using the Sauce Connect orb in your workflows, refer to the [Sauce Connect Orb](https://circleci.com/developer/orbs/orb/saucelabs/sauce-connect) page in the [CircleCI Orbs Registry](https://circleci.com/developer/orbs).
+
+### saucectl
+{: #saucectl }
+
+If you're using JavaScript to test your web application, you can still take advantage of the Sauce Labs platform by using [`saucectl`](https://docs.saucelabs.com/testrunner-toolkit) with the JS framework of your choice, and then integrating the [saucectl-run Orb](https://circleci.com/developer/orbs/orb/saucelabs/saucectl-run) in your CircleCI workflow.
+
+1. Add your `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY` as Environment variables in your Circle CI project.
+2. Modify your CircleCI project `config.yml` to include the saucectl-run orb and then call the orb as a job in your workflow.
+
+{% raw %}
+```yaml
+version: 2.1
+orbs:
+  saucectl: saucelabs/saucectl-run@1.0.0
+
+jobs:
+  test-cypress:
+    docker:
+      - image: circleci/node:12.21
```

Comment:
Recommending updating docker images to use our next-gen images: https://circleci.com/developer/images/image/cimg/node

An aside, it may be difficult to keep this example in-sync with orb updates. Orbs themselves hold usage examples as well. I think it might be worth considering linking to an example in the orb rather than including it here.

Reply:
Thanks for the feedback, @KyleTryon. I am going to circle back with the dev team to see if the orb has an example we would prefer to reference here and make sure the docker image is current.

Reply:
@KyleTryon  is there any update to be done to the orb itself, or that's just this example and our examples that needs to be updated ?

Reply:
@KyleTryon Any more feedback here? I think it's just the image used that needs updating but wanted to check in with you to confirm :-)

Reply:
Hey, could we just update to use a next gen image for this example then we can get these changes merged, and as you say move the wider orb update conversation outside this PR?

Reply:
Thanks @rosieyohannan! I have made the update to the image reference. Our developer has also tested the orb using the latest image to verify it works and has made the changes to our examples in the circleCI repo.